rllib/tuned_examples/appo/pong-appo.yaml

```diff
@@ -12,10 +12,13 @@ pong-appo:
     config:
         # Works for both torch and tf.
         framework: torch
+        # Make analogous to old v4 + NoFrameskip.
         env_config:
             frameskip: 1  # no frameskip
-        vtrace: True
-        use_kl_loss: False
+            full_action_space: false
+            repeat_action_probability: 0.0  # deterministic
+        vtrace: true
+        use_kl_loss: false
```

Comment:
do we need to disable the kl loss? Is this something that we were doing in the old stack?

Reply:
I noticed in the new benchmarks that KL-loss might destabilize the run, but this observation was made while several other issues had not been fixed so it could have been a red herring. Either way, we don't currently use kl_loss in our ols-stack benchmark, so I would like to keep it switched off until we have fully investigated a possible benefit of using this term.